src/main/deploy/pathplanner/paths/New Path.path

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Have we tested this and verified it works?

Reply:
No. Not sure what it is. Got committed but we need the vision additions from drivetrain.

Reply:
This can be deleted for now. It's just the default path name from the GUI and shows up once synced to a project.